contracts/TrueCurrencies/TrueRewardBackedToken.sol

```diff
@@ -81,16 +72,29 @@ contract TrueRewardBackedToken is RewardTokenWithReserve {
     function totalSupply() public view returns (uint256) {
         // if supply in opportunity finOp, return value including finOp value
         // otherwise call super to return normal totalSupply
-        if (opportunitySupply() != 0) {
-            // calculate depositToken value of finOp total supply
-            uint depositValue = _toToken(opportunitySupply(), opportunity());
-
+        if (opportunityRewardSupply() != 0) {
             // return token total supply plus deposit token value
```

Comment:
This comment is somewhat misleading because we use 'deposit' to refer to the other thing...